src/workspacer.Shared/Workspace/IWorkspaceContainer.cs

```diff
@@ -16,6 +16,8 @@ public interface IWorkspaceContainer
 
         IWorkspace GetNextWorkspace(IWorkspace currentWorkspace);
         IWorkspace GetPreviousWorkspace(IWorkspace currentWorkspace);
+        int GetNextWorkspaceIndex(IWorkspace currentWorkspace);
```

Comment:
Since there's no guarantee that this method will be called with the _actual_ current workspace, I personally think that it would make more sense to use a name like `workspace` or `targetWorkspace`.

Similarly for `GetPreviousWorkspaceIndex`.